src/Layout/Presentation/SingleDistrict/InfoBox.tsx

```diff
@@ -16,7 +16,7 @@ export class InfoBox extends React.Component<InfoBoxProps> {
                       " hadde nærmeste kvotient og trengte ",
                       vulnerable.moreVotesToWin,
                       " flere stemmer for å vinne mandatet. ",
-                          vulnerableVotes.partyCode,
+                    <abbr title={vulnerableVotes.partyName} key={vulnerableVotes.partyCode}>{vulnerableVotes.partyCode}</abbr>,
```

Comment:
You appear to have solved issue #190 without assigning it to yourself or to this PR.

Reply:
Ok my bad. We have different solutions for adding the partyName to the abbreviation. Which one should we go for?

Reply:
The goal is to migrate away from API v1, and API v3 does not pass PartyName with the results. So it would be preferrable to implement it using the party map from API v3, as I have done.